### PR TITLE
Add Bool::convert()

### PR DESCRIPTION
**Pull request by chadicus**

### CODE REVIEW

**Comment by chrisryan** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -169,6 +169,14 @@ The following example converts `$value` to a boolean allowing the strings "on" a
 ```php
 $enabled = \DominionEnterprises\Filter\Booleans::filter($value, false, ['on'], ['off']);
 ```
+#### Bool::covert
+Aliased in the filterer as `bool-covert`, this filter will covert a given boolean value into the provided true or false conditions. By default the
```
Comment (chrisryan) [inline README.md]: doc needs to be updated to match new class name. PR title update would be good to for history.

File: README.md [hunk 2]
```diff
@@ -169,6 +169,14 @@ The following example converts `$value` to a boolean allowing the strings "on" a
 ```php
 $enabled = \DominionEnterprises\Filter\Booleans::filter($value, false, ['on'], ['off']);
 ```
+#### Bool::covert
```
Comment (nubs) [inline README.md]: those are some sneaky booleans.  `s/covert/convert/` ;)

File: README.md (hunk 1 above)
Comment (nubs) [inline README.md]: ditto here too

File: README.md [hunk 3]
```diff
@@ -169,6 +169,14 @@ The following example converts `$value` to a boolean allowing the strings "on" a
...
+Aliased in the filterer as `bool-covert`, this filter will covert a given boolean value into the provided true or false conditions. By default the
+return values are the strings 'true' and 'false'
+
+The following coverts the boolean `$value` to either 'yes' or 'no'
```
Comment (nubs) [inline README.md]: and here

**Comment by chrisryan** [comment]
:+1:

**Pull request merged**